mkosi/util.py

```diff
@@ -224,6 +224,9 @@ def _generate_next_value_(name: str, start: int, count: int, last_values: Sequen
     def values(cls) -> list[str]:
         return list(map(str, cls))
 
+    def to_json_repr(self) -> str:
+        return cast(str, self.value)
```

Comment:
Bit confused why we need this? Can't we just check for isinstance of StrEnum in the json encoder? I think I'd prefer that